<launch>
  <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="mob_plat_joy">
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>
  <!-- Axes -->
  <param name="axis_accel" value="4" type="int"/>
  <param name="axis_brake" value="3" type="int"/>
  <param name="axis_turn" value="0" type="int"/>
  <param name="scale_accel" value=".1" type="double"/>
  <param name="scale_turn" value="0.4" type="double"/>
  <param name="stop_button" value="1" type="int"/>
  <node pkg="mobile_platform_joy" type="mob_plat_teleop_joy" name="teleop"/>
</launch>